docs/pages/enroll-resources/kubernetes-access/rancher-integration.mdx

```diff
@@ -0,0 +1,269 @@
+ 
+This guide demonstrates how to configure Teleport's Kubernetes RBAC to grant access using Rancher's built-in or custom roles — such as the Cluster Member role. 
+This approach enables you to manage Kubernetes access through Teleport while aligning with Rancher’s existing RBAC structure.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- Access to a Rancher UI Cluster with admin privileges
+- Existing cluster(s) managed through Rancher
+- Rancher RBAC already configured with users/groups assigned appropriate roles (such as Cluster Member)
+- (Optional) Rancher authentication configured with external authentication providers
+
+Within Rancher, cluster and project roles are implemented on top of [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+Therefore, enforcement of permissions and roles is performed by Kubernetes.
+
+## Step 1/4. Discover existing Rancher RBAC configuration
+
+This guide assumes you already have Rancher RBAC configured with users/groups assigned appropriate roles (such as Cluster Member). 
+In this step, we'll discover your existing Rancher RBAC configuration to align Teleport with your current access patterns.
+
+ ![Kubernetes access architecture](../../../img/k8s/rancher-cluster-member.png)
+
+To proceed, you'll need to:
+
+1. Authenticate to your cluster with administrative privileges. See guide to [Access a Rancher cluster with kubectl](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig)
+
+2. Discover Rancher RBAC bindings and identifiers:
+- Navigate to **Cluster** in left sidebar
+- Select **More Resources**, Navigate to the **RBAC** tab
+- On the Rbac Tab, select Rolebindings, you'll be presented witha list of Role/Rolebindings with the Kubernetes Rbac subject identifiers.
+
+ ![Kubernetes access architecture](../../../img/k8s/rancher-rbac.png)
+
+Alternatively, you can run the command below to list Kubernetes RoleBindings and extract the group information that Teleport will use to match your existing Rancher access patterns:
+
+   ```code
+   # List all rolebindings with their names and associated groups
+   $ kubectl get rolebindings -n <namespace> -o custom-columns='NAME:.metadata.name,ROLE:.roleRef.name,GROUPS:.subjects[?(@.kind=="Group")].name'
+   
+   # Example output:
+     NAME                     ROLE            GROUPS
+     crtb-w8l2g-cluster-member cluster-member github_team://7415717
+   ```
+Note: This example uses GitHub as the external authentication provider. If your Rancher setup uses a different [external authentication service](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config#external-vs-local-authentication),
```

Comment:
In the steps of a how-to guide, we generally try to avoid taking the reader outside of the page so they can follow the instructions as quickly as possible. We use a "Further reading" section for more context. 

If the user must have an external authentication provider for this to work, I would add that as a prerequisite and link to this guide.